Makefile

```diff
@@ -95,14 +100,15 @@ dependency: ## Dependency maintanance
 	go mod tidy
 
 .PHONY: dockerimage
-dockerimage: build ## Build docker image
-	docker build --no-cache --build-arg HELM_VERSION=$(HELM_VERSION) -t $(DOCKER):$(VERSION) -f AlpineTest.Dockerfile .
+dockerimage: build-amd64 ## Build docker image
+	docker build --no-cache --build-arg HELM_VERSION=$(HELM_VERSION) --build-arg BUILDPLATFORM=amd64 -t $(DOCKER):$(VERSION) -f AlpineTest.Dockerfile .
```

Comment:
It seems like the test image has to specifically be built and run as a linux/amd64 binary in order to work, so, I do that here.